manifests/params.pp

```diff
@@ -475,6 +475,15 @@
   # this switch also controls Ruby profiling, by default disabled for Puppetserver 2.x, enabled for 5.x
   $server_puppetserver_metrics = versioncmp($::puppetversion, '5.0.0') >= 0
 
+  # Puppetserver metrics shipping
+  $server_metrics_jmx_enable        = true
+  $server_metrics_graphite_enable   = true
```

Comment:
I'm unsure about this. Doesn't that mean, per default puppetserver will try to send its data to that graphite host?

Reply:
I think you are right. Changed $server_metrics_graphite_enable default value to false
 and left $server_metrics_jmx_enable unchanged. Now these settings are same as defaults in original metrics.conf.